src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+import { Box, Button, Flex, Icon, Link, Text } from '@chakra-ui/react'
+import SidebarList from './SidebarList'
+import { IoIosSettings } from 'react-icons/io'
+
+const Sidebar = () => {
+  const sidebarHeight = Math.round(window.innerHeight) - 75
+  const [isStorageActive, setIsStorageActive] = useState(false)
+  const [isListActive, setIsListActive] = useState(true)
+  return (
+    <Box
+      position="fixed"
+      top="0"
+      right="0"
+      h={sidebarHeight + 'px'}
```

Comment:
Wydaje mi się, że nie ma konieczności zasysania wysokości okna w JS. [Takie coś](https://stackoverflow.com/questions/69572406/chakra-ui-full-height-component) powinno styknąć. Ale to szczegół,  ogólnie ok.